package.json

```diff
@@ -62,15 +62,15 @@
     "@commitlint/config-conventional": "^18.4.3",
     "@eslint-community/eslint-plugin-eslint-comments": "^4.1.0",
     "@eslint/js": "^8.55.0",
-    "@graphql-eslint/eslint-plugin": "^3.20.0",
+    "@graphql-eslint/eslint-plugin": "^3.20.1",
     "@prettier/plugin-pug": "^3.0.0",
-    "@types/eslint": "^8.44.7",
+    "@types/eslint": "^8.56.0",
     "@types/prettier-linter-helpers": "^1.0.1",
     "commitlint": "^18.4.3",
-    "eslint": "^8.44.0",
+    "eslint": "^8.56.0",
```

Comment:
Are all these package changes required? Tests seem to pass with eslint ^8.44

Reply:
I was checking whether `context.languageOptions` is available in `eslint` and `@types/eslint` and sadly it's unavailable in types yet.

I can discard such related changes into other PRs, but I think they are just tiny enough though.